.vscode/settings.json

```diff
@@ -1,2 +1,16 @@
 {
+    "workbench.colorCustomizations": {
```

Comment:
Why are you commiting your .vscode/settings.json but you ignore .vscode/ folder ?

Reply:
Good question.